### DISCUSSION TITLE
Official Instance Outages

### DISCUSSION THREAD

**Discussion by spikecodes** [spikecodes/libreddit#756] [Announcements]
Hi,

Libreddit's primary official instance (libreddit.spike.codes) was accidentally flagged as in violation of Replit's TOS and is seeing a temporary outage because of this.

The team is discussing long term hosting options and will be open to suggestions/feedback in this thread.

Thank you for your patience and please use one of the many community instances in the meantime.

-Spike

**Comment by avincent98144** [comment]
I have not used SPIKE.CODES for over a year and switched to BUS-HIT.ME and in use successfully since - until March 14, 2023 - 1:29:09 AM PDT when an error "508 BAD GATEWAY" was presented any time I added a FILTER command. It did not show an error prior to yesterday and now shows in all FILTER commands since. All other BUS-HIT.ME instance features are working as expected - even just now March 15, 2023 - 2:30:52 PM PDT.

**Reply by avincent98144** [reply]
March 18, 2023 - 5:19:23 AM PDT: Issue was resolved within 24 hours [6 days ago] and as of March 18, 2023 - 5:19:51 AM PDT the issue has re-appeared.

**Comment by avincent98144** [comment]
NOTE: I have tried 4 other instances, all with the same exact issue.

**Comment by spikecodes** [comment]
Hi Anthony. Can you open a Bug Report issue for this? I don't believe it pertains to the official instance's outage, sounds more like a Libreddit code error.

**Reply by avincent98144** [reply]
I believe I did that on my first post.

**Comment by lhhel9l3** [comment]
Hi, is there a way to restore our config from the official instance when it is down, and transfer it to a different instance? For example, is it stored in a cookie somewhere?

**Comment by spikecodes** [comment]
Hi, I moved libreddit.spike.codes to a different repo which should act as a temporary fix (may require force refreshes every so often) so that people can access their config (stored as cookies) and back them up elsewhere.

**Comment by Traction6131** [comment]
Would it just be a possiblility to use a vps or even free hosting solutions like oracle cloud or vultr? For regions, Canada (montreal) seems good, since my Instance (snoo.habedieeh.re) has never suffered Issues with reddit's CDN. Of course one could consider having multiple locations as well (e.g. via anycast or geoDNS). Lastly, there are also free hosting options for open source projects, might also want to check that out.
